README.md

```diff
@@ -897,6 +898,32 @@ If you're curious how this could ever be useful, have a look at the [Kinto formb
 
 Props passed to a custom SchemaField are the same as [the ones passed to a custom field](#field-props).
 
+### Customizing any fields or widgets
+
+You can override any default field and widget, including the internal widgets like the `CheckboxWidget` that `ObjectField` renders for boolean values. You can override any field or widget just by providing the customized fields/widgets in the `fields` and `widgets` props:
+
+```jsx
+
+const CustomCheckbox = function(props) {
+  return (
+    <button id="custom" className={props.value ? "checked" : "unchecked"} onClick={props.onChange(!props.value)}>
+    	{props.value}
+    </button>
+  );
+};
+
+const widgets = {
+  CheckboxWidget: CustomCheckbox
+};
+
+render((
+  <Form schema={schema}
+        uiSchema={uiSchema}
+        formData={formData}
+        widgets={widgets} />
+), document.getElementById("app"));
```

Comment:
It may look super obvious to seasonned react developers, but maybe we should augment this section with an example of a reusable customized form class, eg.

```jsx
const fields = {StringField: CustomString};
const widgets = {CheckboxWidget: CustomCheckbox};

function MyForm(props) {
  return <Form fields={customFields} widgets={customWidgets} {...props} />;
}

render((
  <MyForm schema={schema}
    uiSchema={uiSchema}
    formData={formData} />
), document.getElementById("app"));
```